lib/livebook/application.ex

```diff
@@ -393,6 +418,8 @@ defmodule Livebook.Application do
     })
   end
 
+  # ELIXIR_ERL_OPTIONS used to configure EPMD module in releases.
+  defp config_env_var?("ELIXIR_ERL_OPTIONS"), do: true
```

Comment:
We need to delete this, otherwise any Elixir executable invoked within the runtime will have the custom EPMD configuration.

On the flip side, this means if the user has set `ELIXIR_ERL_OPTIONS` somewhere, it will no longer propagate to the runtime (like `ERL_AFLAGS` and `ERL_ZFLAGS` currently propagate). Is this acceptable?

Another option is to not delete `ELIXIR_ERL_OPTIONS` here, then it will be also available in the runtime (so we don't need to set it by hand), and then we delete it from the runtime instead.

So overall, we need to answer which of `ERL_AFLAGS`, `ERL_ZFLAGS` and `ELIXIR_ERL_OPTIONS` we want to pass all the way down, from Livebook -> Runtime -> Executable, or if we want to delete some (or all) of them along the way (and if so, where).

Reply:
@josevalim we can set `ELIXIR_ERL_OPTIONS_WAS` or similar, and just revert the value here, so whatever the user sets gets propagated as expected, wdyt?

Reply:
I thought about that but I also think it is useful to have a value that only applies to Livebook but not the runtimes and `ELIXIR_ERL_OPTIONS` sounds like a good candidate. We just have to document it?

Reply:
We don't expect ERL_AFLAGS to be set it either, but it just works by the nature of things, it that makes any sense.

Reply:
What I mean is that Livebook-specific env var is probably not interesting for the end user, only us, so we can replicate that without affecting how envs work for the user. Again, no strong opinion though.